index.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     'prefer-template': 'warn',
     'react/prop-types': 'error',
     'react-hooks/exhaustive-deps': 'error',
+    'react/jsx-key': ['warn', { checkFragmentShorthand: true }],
```

Comment:
As this option has at the moment a default value of `false`, but is going to change for `true` in new major version, better have it already to `true`.
![CleanShot 2022-08-26 at 16 43 45](https://user-images.githubusercontent.com/6507291/186930666-07177829-326f-4298-a8e4-87e18ebeff7e.png)

Reply:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md